charts/karpenter-crd/artifacthub-repo.yaml

```diff
@@ -0,0 +1,7 @@
+repositoryID: fda7ffc4-4672-4218-8264-321ec3b4e3cc
+owners: []
+#   - name: awsadmin1
+#     email: artifacthub1@aws.com
+ignore:
+  - name: karpenter-crd
+    version: (?:^\d+$)|(?:^v?0\.0\.0)|(?:^v?\d+\-)
```

Comment:
Can we add a comment here to indicate why we want to ignore ArtifactHub indexing these versions? It's not immediately apparent to me why we are ignoring these versions just reading through it.

Reply:
Am I right in thinking that this repo should only ever get stable versions now? If so then this is to exclude old non-stable versions. This could be solved without the regex by just removing all non-stable versions.

Reply:
Got it. So this was produced by looking through all of the tags in our repository for the chart? What about the tags for chart signatures? Do we also need to ignore these? I believe that they start with `sha256-`

Reply:
Only the Helm chart OCI media types are processed so signatures are fine, it's just the old non-production charts which need ignoring (the `yyyyddmm` ones actually error).